README.md

```diff
@@ -23,13 +23,15 @@ Depending on network you want to join you can use appropriate launcher, all can
 
 ### Building the client
 
-#### Prerequisites to build
+#### SBT
+
+##### Prerequisites to build
 
 - JDK 1.8 (download from [java.com](http://www.java.com))
```

Comment:
btw, i think now is working ok with JDK 11

Reply:
It is. Do we all want to jump into 11?

Reply:
who knows hehe..